style.css

```diff
@@ -0,0 +1,107 @@
+* {
+    margin: 0;
+}
+html{
+    scroll-behavior: smooth;
+}
+body{
+    background-color: #29058f;
+    min-height: 100vh;
+    color: #abcdef;
+}
+.navbar{
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100%;
+    display: flex;
+    align-items: center;
+    background-color: #111111f2;
+    padding: 20px;
+    z-index: 100;
+}
+.name{
+    flex: 0.6;
+    font-size: 22px;
+}
+.link{
+    flex: 0.4;
+    font-size: 22px;
+}
+a{
+    text-decoration: none;
+    color: #fff;
+    padding: 20px;
+}
+a:hover{
+    background-color: #333;
+}
+@media (max-width: 1000px) {
+    .namw{
+        flex: 0.4;
+    }
```

Comment:
on this line of CSS, I think you meant to type name instead of namw. I would recommend correcting that issue, so the HTML and CSS can work together 👍

Reply:
- [ ] Well, you have respected some requirements but`me` as the student, I don't understand what you mean by this, what if `namw` was the class declared in the HTML? if you figure out that this class is misspelled, then you should tell the student to fix the type because the CSS code is not loaded, you should give a refference to the HTML class and the output(the result of the app in the browser to show that the CSS is not loaded because of the type)